docs/detections/alerts-view-details.asciidoc

```diff
@@ -6,111 +6,169 @@
+
+* **Analyzer preview**: Shows a preview of the <<visual-event-analyzer,visual analyzer graph>>. The preview displays up to three levels of the analyzed event's ancestors and up to three levels of the event's descendants and children. The ellipses symbol (**`...`**) indicates the event has more ancestors and descendants to examine. Click **Analyzer preview** to open the **Event Analyzer** tab in Timeline.
+
+[discrete]
+[[insights-section]]
+== Insights 
+
+The Insights section is located on the **Overview** tab in the right panel. It offers different perspectives from which you can assess the alert. Click **Insights** to display overviews for <<entities-overview,related entities>>, <<threat-intelligence-overview,threat intelligence>>, <<correlations-overview,correlated data>>, and <<prevalence-overview,host and user prevalence>>.  
+
+[role="screenshot"]
+image::images/insights-section-rp.png[Insights section of the Overview tab, 65%]
+
+[discrete]
+[[entities-overview]]
+=== Entities 
+
+The Entities overview provides high-level details about the user and host that are related to the alert. Host and user risk classifications are also available with a https://www.elastic.co/pricing[Platinum subscription] or higher. 
+
+NOTE: <<user-risk-score,User>> and <<host-risk-score,host>> risk scores are technical preview features. 
+
+[role="screenshot"]
+image::images/entities-overview.png[Overview of the entity details  section in the right panel, 60%]
 
 [discrete]
-[[alerts-enrich-ti]]
-==== Alerts enriched with threat intelligence
-Alerts can be enriched with contextually relevant threat intelligence that you can use for triaging and investigating alerts. If available, this information appears in the Enriched data section.
+[[expanded-entities-view]]
+==== Expanded entities view
+
+From the right panel, click **Entities** to open a detailed view of the host an user associated with the alert. The expanded view also includes risk scores and classifications (if you have a Platinum subscription or higher) and activity on related hosts and users. 
+
+[role="screenshot"]
+image::images/expanded-entities-view.png[Expanded view of entity details, 70%]
+
+[discrete]
+[[threat-intelligence-overview]]
+=== Threat intelligence 
+
+The Threat intelligence overview shows matched indicators, which provide contextually relevant threat intelligence on the alert. 
 
-When gathering threat intelligence data for an alert, {elastic-sec} queries the alert for indicator matches from the past 30 days. The query looks for the following fields:
+[role="screenshot"]
+image::images/threat-intelligence-overview.png[Overview of threat intelligence on the alert, 70%]
+
+To gather threat intelligence on an alert, {elastic-sec} queries alert documents from the past 30 days and searches for fields that contain known threat intelligence. {elastic-sec} looks for the following fields:
```

Comment:
Have a bit of confusion, because this section related to 

**Fields enriched with threat intelligence**

But currently it looks like, that if for both **Fields enriched with threat intelligence** and **Threat match detected**


Another confusion is that for **Expanded threat intelligence view**, we have the same mechanism, but their described a bit differently.

But it the same things, in for enrichments, only difference that in this section we show how much enrichment, and on other we show it in details

Reply:
@nikitaindik to clarify, are you saying that this information only applies to non-indicator match alerts that have fields enriched with threat intelligence? Or, does this information apply to all alert types (i.e., indicator match alerts and non-indicator match alerts) that contain threat intelligence?

Reply:
wrong Nikita in the handle:)

Yes. this describe hot real-time enrichments works, but not IM rule

Because for IM rule you can specify other time range and other fields, and then it persist enrichment info in alert document

Reply:
Oops! Sorry about that 😅 

Ok, I'll move this information to the correct spot then. Thank you!